.github/workflows/test_app.yml

```diff
@@ -95,6 +95,8 @@ jobs:
         with:
           chromedriver-version: ${{inputs.chrome_version}}
         name: Install Chrome version ${{inputs.chrome_version}}
+      - run: |
```

Comment:
Seems like there was a bad merge and we have lots of noise in this PR now, can you check it out @alecslupu ?